src/utils/testHelpers.js

```diff
@@ -0,0 +1,31 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { initialState } from '../store/modules/auth';
+
+export const findByTestAttribute = (component, attribute) => {
+  const wrapper = component.find(`[data-test='${attribute}']`);
+  return wrapper;
+};
+
+export const getLocation = () => ({
```

Comment:
we can move this back to the test files.

Reply:
Done